src/runtime/profiling.cc

```diff
@@ -878,6 +878,10 @@ PackedFunc WrapTimeEvaluator(PackedFunc pf, Device dev, int number, int repeat,
     // skip first time call, to activate lazy compilation components.
     pf.CallPacked(args, &temp);
 
+    // allocate two large arrays to flush L2 cache
+    NDArray arr1 = NDArray::Empty({256 * 1000000 / 4}, {kDLInt, 32, 1}, dev);
```

Comment:
How about we add a parameter, L2 flush_size, which enables us to choose to do so or not, and when it is 0 we skip